src/github.com/getlantern/idletiming/idletiming_conn.go

```diff
@@ -132,14 +139,18 @@ func (c *IdleTimingConn) Write(b []byte) (int, error) {
 		maxDeadline := time.Now().Add(c.halfIdleTimeout)
 		if writeDeadline != epoch && !maxDeadline.Before(writeDeadline) {
 			// Caller's deadline is before ours, use it
-			c.conn.SetWriteDeadline(writeDeadline)
+			if err := c.conn.SetWriteDeadline(writeDeadline); err != nil {
+				log.Debugf("Unable to set write deadline: %v", err)
+			}
 			n, err := c.conn.Write(b)
 			c.markActive(n)
 			totalN = totalN + n
 			return totalN, err
 		} else {
 			// Use our own deadline
-			c.conn.SetWriteDeadline(maxDeadline)
+			if err := c.conn.SetWriteDeadline(maxDeadline); err != nil {
+				log.Debugf("Unable to set write deadline: %v", err)
```

Comment:
I'm torn about these ones. Maybe error too? I actually think they fail all the time though because enproxy doesn't support these deadline funcs, but it's actually still unclear to me why that's not a real problem (or maybe it actually is a real bug, not sure).